Makefile

```diff
@@ -57,7 +57,7 @@ licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
 	golicense hack/license-config.hcl karpenter
 
 apply: ## Deploy the controller from the current state of your git repository into your ~/.kube/config cluster
-	helm upgrade --install karpenter charts/karpenter --namespace karpenter \
+	helm upgrade --create-namespace --install karpenter charts/karpenter --namespace karpenter \
```

Comment:
I don't think this should matter, but in our getting-started guide we do it like this `--namespace karpenter --create-namespace`. Can you make sure that the order doesn't matter?

Reply:
yep, order doesn't matter:

```
Usage:
  helm upgrade [RELEASE] [CHART] [flags]
```